deploy/crs/v2/snippets/common_config.yaml

```diff
@@ -12,3 +12,6 @@ spec:
     # By default is the cr.Name
     applicationName: other
     adminPassword: example
+    # startupStrategy could be OpenShiftStartupStrategy or ControllerBasedStartupStrategy
```

Comment:
do not edit the existing one, please  create a new snippet.

Reply:
ok